src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -418,6 +426,106 @@ public void init(AdaptorContext context) {
       }
     }
 
+    // This URL is used for search and download.
+    this.indexingContentServerUrl =
+        config.getValue("opentext.indexing.contentServerUrl");
+    if (this.indexingContentServerUrl.isEmpty()) {
+      this.indexingContentServerUrl = this.contentServerUrl;
+    }
+    log.log(Level.CONFIG, "opentext.indexing.contentServerUrl: {0}",
+        this.indexingContentServerUrl);
+
+    try {
+      this.readTimeout = Integer.parseInt(
+          config.getValue("adaptor.docContentTimeoutSecs")) * 1000;
+    } catch (NumberFormatException e) {
+      throw new InvalidConfigurationException(
+          "Invalid value for adaptor.docContentTimeoutSecs: "
+          + config.getValue("adaptor.docContentTimeoutSecs"));
+    }
+    try {
+      this.connectTimeout = Integer.parseInt(
+          config.getValue("adaptor.docHeaderTimeoutSecs")) * 1000;
+    } catch (NumberFormatException e) {
+      throw new InvalidConfigurationException(
+          "Invalid value for adaptor.docHeaderTimeoutSecs: "
+          + config.getValue("adaptor.docHeaderTimeoutSecs"));
+    }
+
+    // Don't log these parameters unless they're set to something
+    // other than the default.
+    this.downloadMethod = config.getValue("opentext.indexing.downloadMethod");
+    if (!this.downloadMethod.equals("contentserver")) {
+      if (!(this.downloadMethod.equals("cws")
+              || this.downloadMethod.equals("cwscontenthandler"))) {
+        throw new InvalidConfigurationException(
+            "Invalid value for opentext.indexing.downloadMethod");
+      }
+      log.log(Level.CONFIG, "opentext.indexing.downloadMethod: {0}",
+          this.downloadMethod);
+      if (this.downloadMethod.equals("cwscontenthandler")) {
+        this.downloadHandlerUrl =
+            config.getValue("opentext.indexing.downloadHandlerUrl");
+        if (this.downloadHandlerUrl.isEmpty()) {
+          throw new InvalidConfigurationException(
+              "opentext.indexing.downloadHandlerUrl is missing");
+        } else {
+          log.log(Level.CONFIG, "opentext.indexing.downloadHandlerUrl: {0}",
+              this.downloadHandlerUrl);
+        }
+      }
+    }
+
+    // This URL is used for search and download.
```

Comment:
This code got added twice, above and below. I didn't compare the two copies, so you should probably make sure they're identical before deleting one.